### ISSUE TITLE
re-implementation with tensorflow cannot achieve the same accuracy in mxnet.

### ISSUE THREAD

**Issue by Jar7** [deepinsight/insightface#935]
Hi, I apply insightface for face recognition with my own dataset by simply add 2 fc layers and output the index of the class.
Firstly I trained it on mxnet, fixed the feature network(pretrained model: r100), only finetune that 2 additional fc layers. Then I re-implemented it with tensorflow, the parameters and optimizer and initialization is the same. However, I cannot reproduce the same result as on mxnet.
Any suggestion, please? Is there any detail could I ignored?
Thank you.
